src/cpp/database/database_queue.cpp

```diff
@@ -380,11 +380,11 @@ EntityId DatabaseEntityQueue::process_endpoint_discovery(
 
     if (info.kind() == EntityKind::DATAREADER)
```

Comment:
The if statement is redundant, since the same action is performed independently of the condition. Please remove.

```Suggestion
 name << info.topic_name << "_" << info.guid.entityId;
```